docs/faq.rst

```diff
@@ -660,7 +663,8 @@ it would be much faster.
 Another possible reason is that files don't always have the same path, for
 example if you mount a filesystem without stable mount points for each backup or if you are running the backup from a filesystem snapshot whose name is not stable.
 If the directory where you mount a filesystem is different every time,
-Borg assume they are different files.
+Borg assume they are different files. This is true even if you backup these files with relative pathnames -- borg uses full
```

Comment:
while you are at it, please fix that typo also: "assumes"

do we use `--` for long dashes elsewhere, too?

Reply:
Seems not. Ameded to `-`